addons/mass_mailing/models/mailing_trace.py

```diff
@@ -106,34 +97,15 @@ class MailingTrace(models.Model):
         ("mail_optout", "Opted Out"),
         ("mail_dup", "Duplicated Email"),
     ], string='Failure type')
-    state_update = fields.Datetime(compute="_compute_state", string='State Update',
-                                   help='Last state update of the mail',
-                                   store=True)
+    # Link tracking
+    links_click_ids = fields.One2many('link.tracker.click', 'mailing_trace_id', string='Links click')
+    links_click_done = fields.Boolean('Links Clicked', help='If customer clicked on at least one tracked link')
```

Comment:
Feels strange that those 2 are not related in any way, seems like one should be computed based on the other?

Reply:
It is computed by hand (aka updated when going through link click code). Purpose is to avoid having a computed based on o2m, quickly inefficient when dealing with a model with high volume.

Reply:
I made the same move for status, it is actually one of the main reason of this PR: simpler to update manually as we go through some flows, no need to keep computed fields as people should not update "dependencies" manually.